packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-field-input/question.js

```diff
@@ -117,6 +137,45 @@ class MultiFieldInputQuestion extends Question {
 			}
 		];
 	}
+
+	/**
+	 * returns the formatted answers values to be used to build task list elements
+	 * @param {Journey} journey
+	 * @param {InputField[]} inputFields
+	 * @returns {String}
+	 */
+	formatContactDetails(journey, inputFields) {
+		const firstNameField =
+			inputFields.find((inputField) => inputField.fieldName.includes('FirstName'))?.fieldName || '';
+		const lastNameField =
+			inputFields.find((inputField) => inputField.fieldName.includes('LastName'))?.fieldName || '';
+		const companyNameField =
+			inputFields.find((inputField) => inputField.fieldName.includes('CompanyName'))?.fieldName ||
+			'';
+
+		if (!journey.response.answers[firstNameField]) return this.NOT_STARTED;
+
+		const contactName = `${journey.response.answers[firstNameField]} ${journey.response.answers[lastNameField]}`;
+		const companyName = journey.response.answers[companyNameField];
+
+		return contactName + (companyName ? `<br>${companyName}` : '');
```

Comment:
we need this to be escaped for if a user enters html into one of the fields
may want to use the escape function: const escape = require('escape-html');

Reply:
I've updated to use escape and nl2br